docs/en/02_embaro_expiry.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Embargo and Expiry
+summary: Configure embargo and expiry functionality for content changes
+icon: calendar-alt
+---
+
+# Embargo and expiry
+
+This add-on functionality allows you to embargo some content changes to only appear as published at some future date. To enable it,
+add the [`WorkflowEmbargoExpiryExtension`](api:Symbiote\AdvancedWorkflow\Extensions\WorkflowEmbargoExpiryExtension).
+
+```yml
+SilverStripe\CMS\Model\SiteTree:
+  extensions:
+    - 'Symbiote\AdvancedWorkflow\Extensions\WorkflowEmbargoExpiryExtension'
+```
+
+Make sure the [Queued Jobs](https://docs.silverstripe.org/en/optional_features/queued-jobs/) module is installed and configured correctly.
```

Comment:
```suggestion
Make sure the [Queued Jobs](https://docs.silverstripe.org/en/optional_features/queuedjobs/) module is installed and configured correctly.
```
According to https://github.com/silverstripe/doc.silverstripe.org/pull/331

Reply:
Updated